openmls/src/group/public_group/process.rs

```diff
@@ -225,9 +227,35 @@ impl PublicGroup {
                 ))
             }
             Sender::External(_) => {
-                // We don't support messages from external senders yet
-                // TODO #151/#106
```

Comment:
Can you leave the TODO, since external adds are not done yet?

Reply:
I left them. They're just down in the code, after the processing of the external remove.